### ISSUE TITLE
Existence of temp files in _meta directory causes CLI to break

### ISSUE THREAD

**Issue by assaflavie** [serverless/serverless#578]
Create a *non* JSON file in `_meta/variables` called `s-variables-dev.json.swp`, fill it with some random text.

Now `sls env list` returns:
```
/usr/local/lib/node_modules/serverless/node_modules/bluebird/js/main/async.js:43
        fn = function () { throw arg; };
                           ^

SyntaxError: Unexpected token a
    at Object.parse (native)
    at Object.exports.readAndParseJsonSync (/usr/local/lib/node_modules/serverless/lib/utils/index.js:520:15)
    at /usr/local/lib/node_modules/serverless/lib/ServerlessMeta.js:52:37
From previous event:
    at ServerlessMeta.load (/usr/local/lib/node_modules/serverless/lib/ServerlessMeta.js:40:25)
    at /usr/local/lib/node_modules/serverless/lib/ServerlessState.js:39:27
    at processImmediate [as _immediateCallback] (timers.js:383:17)
From previous event:
    at ServerlessState.load (/usr/local/lib/node_modules/serverless/lib/ServerlessState.js:38:8)
    at Serverless.init (/usr/local/lib/node_modules/serverless/lib/Serverless.js:113:23)
    at /usr/local/lib/node_modules/serverless/lib/Serverless.js:143:56
From previous event:
    at Serverless._execute (/usr/local/lib/node_modules/serverless/lib/Serverless.js:132:38)
    at Serverless.actions.(anonymous function) (/usr/local/lib/node_modules/serverless/lib/Serverless.js:382:20)
    at Serverless.command (/usr/local/lib/node_modules/serverless/lib/Serverless.js:351:38)
    at Object.<anonymous> (/usr/local/lib/node_modules/serverless/bin/serverless:16:12)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
    at startup (node.js:136:18)
    at node.js:966:3
```

This is true for other CLI commands and a variety of other JSON config file names/locations.

I basically have to quit Vim every time I want to run the CLI (Vim is the one creating swp files).

**Comment by NickBlow** [comment]
I assume its the same problem as #571

**Comment by ac360** [comment]
Fixed in a95d434d4ed73fac3744ef22dc739fe58f157cbf  Please let me know if this doesn't work for you :)
